python/huggingfaceserver/huggingfaceserver/model.py

```diff
@@ -65,15 +68,16 @@ def __init__(self, model_name: str, kwargs,
             assert world_size == tp_degree, f"TP degree ({tp_degree}) doesn't match available GPUs ({world_size})"
         self.model_id = kwargs.get('model_id', None)
         self.model_dir = kwargs.get('model_dir', None)
-        self.do_lower_case = kwargs.get('do_lower_case', True)
-        self.add_special_tokens = kwargs.get('add_special_tokens', True)
+        self.do_lower_case = not kwargs.get('disable_lower_case', False)
+        self.add_special_tokens = not kwargs.get('disable_special_tokens', False)
```

Comment:
Add default values in parser args in `main.py` instead?

Reply:
The default are set here to facilitate tests.